### PR TITLE
ADCM-2972 Add update audit

### PR DESCRIPTION
**Pull request by winmasta**

### CODE REVIEW

**Comment by winmasta** [comment]
Bender build

**Comment by BenderArenadata** [comment]
Hey, @winmasta! Build has been launched!

File: python/audit/utils.py [hunk 1]
```diff
@@ -86,6 +87,15 @@ def _task_case(task_pk: str, action: str) -> Tuple[AuditOperation, AuditObject]:
     return audit_operation, audit_object
 
 
+def _get_service_name(service: ClusterObject) -> str:
+    if service.display_name:
+        name = service.display_name
+    else:
+        name = str(service)
+
+    return name
```
Comment (a-alferov) [inline python/audit/utils.py:96]: I think this `else` is redundant. And need add `service` to `display_name` or use `service.prototype.name` instead `str(service)`:
```suggestion
def _get_service_name(service: ClusterObject) -> str:
    if service.display_name:
        return service.display_name
    return service.prototype.name
```

**Review by a-alferov — CHANGES_REQUESTED** [review]

File: python/audit/utils.py (hunk 1 above)
Comment (winmasta) [reply python/audit/utils.py]: `service.prototype.name` is nullable so fixed with addition

**Review by a-alferov — APPROVED** [review]

**Pull request merged**